client/scss/components/_promo.scss

```diff
@@ -142,6 +142,10 @@
   left: 0;
   padding: map-get($gutter-width, 'small')/2;
 
+  .promo--on-cream & {
```

Comment:
My thinking was that the promo _itself_ isn't really cream (just the background of the play button) as a result of its surroundings.